fair_drop/suspicious.py

```diff
@@ -0,0 +1,180 @@
+# See https://github.com/convex-labs/honestnft-shenanigans/issues/86 for more details
+# Given a collection of NFTs on OpenSea, detect suspicious NFTs
+import time
+from multiprocessing import Pool
+from argparse import ArgumentParser
+import logging
+
+import cloudscraper
+import pandas as pd
+from bs4 import BeautifulSoup
+from requests.adapters import HTTPAdapter, Retry
+
+
+logging.basicConfig(level=logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument(
+    "-c",
+    "--collection",
+    dest="collection_address",
+    help="Address of the NFT collection",
+    metavar="COLLECTION",
+    required=True,
+)
+parser.add_argument(
+    "-r",
+    "--retries",
+    dest="retries",
+    help="Number of retry attempts",
+    epilog="See https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html",
```

Comment:
Isn't epilog supposed to be used in the ArgumentParser() only and not in the add_argument()?
I'm getting this error:
```
Traceback (most recent call last):
  File "/home/xxx/honestnft-shenanigans/fair_drop/suspicious.py", line 25, in <module>
    parser.add_argument(
  File "/usr/lib/python3.10/argparse.py", line 1434, in add_argument
    action = action_class(**kwargs)
TypeError: _StoreAction.__init__() got an unexpected keyword argument 'epilog'

```

Reply:
I've removed the `epilog` parameter. Got confused in the argparse documentation :sweat_smile: